pkg/controller/configmap/configmap_controller.go

```diff
@@ -196,6 +196,13 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 			}
 		}
 	}
+    networkConfigChanged := true
+    if r.sharedInfo.NetworkConfig != nil {
+        networkConfigChanged = HasNetworkConfigChange(networkConfig, r.sharedInfo.NetworkConfig)
+        if err != nil {
```

Comment:
where do we set `err `? might be irrelevant in context with prev comment

Reply:
removed